app/controllers/heartbeat_controller.rb

```diff
@@ -2,12 +2,20 @@ class HeartbeatController < ApplicationController
   permit_only_with_key
 
   def healthcheck
-    render json: {
-      checks: {
-        sendgrid: sendgrid_alive?,
-        messagelabs: messagelabs_alive?,
-        database: ActiveRecord::Base.connection.active?
-      }
+    database_active = begin
+      ActiveRecord::Base.connection.active?
+    rescue PG::ConnectionBad
```

Comment:
I'm not entirely happy with this dependency on `PG` directly: it could make switching DBs down the line a bit harder. @jasiek / @threedaymonk any thoughts on another class of error I could rescue here?

Also, is there a cleaner way to do this whole block?